### PR TITLE
Add cmf_find_translation twig function documentation

### PR DESCRIPTION
**Pull request by EmmanuelVella**
@dbu Related to https://github.com/symfony-cmf/core-bundle/pull/201

### CODE REVIEW

File: bundles/core/templating.rst [hunk 1]
```diff
@@ -29,6 +29,10 @@ Basic repository operations
 
 * **cmf_find|find(path)**: Get the document for the provided path or null if not found.
   No publish workflow check is done.
+* **cmf_find_translation|findTranslation(($path, $locale, $fallback = true)**: Get the document translation
```
Comment (dbu) [inline bundles/core/templating.rst]: duplicated opening `(`
and can we call the paramter $pathOrDocument to be more explicit?

Comment (EmmanuelVella) [inline bundles/core/templating.rst]: Fixed !

**Comment by dbu** [comment]
sorry, one last thing: we should have a versionadded block after the list of twig functions that says in which version we added this function. otherwise users of an older version will be confused. (for the exact syntax, look for ..versionadded: in the doc sources, we use that in several places)

**Comment by dbu** [comment]
version would be 1.3.1 of the core bundle

**Comment by EmmanuelVella** [comment]
No problem, is it ok like this ?

**Comment by dbu** [comment]
sorry, no the doc should be in the table with the others. the versionadded just explains when this particular function has been added. like "The cmf_find_translation twig function has been added in version 1.3.1 of the CMF CoreBundle."

**Comment by EmmanuelVella** [comment]
Ho sorry, I just fixed it, hope it's ok now !

**Comment by dbu** [comment]
yep, that was the idea. thank you!

**Pull request merged**